src/main/java/interface_adapter/list/ListController.java

```diff
@@ -2,15 +2,25 @@
 
 import use_case.list.ListInputBoundary;
 import use_case.list.ListInputData;
+import use_case.list.removeMedia.RemoveInputBoundary;
+import use_case.list.removeMedia.RemoveMediaInput;
 
 /**
  * Controller for displaying the media collection list.
  */
 public class ListController {
     private final ListInputBoundary listInteractor;
+    private final RemoveInputBoundary removeInteractor;
 
-    public ListController(ListInputBoundary listInteractor) {
+    /**
+     * Constructor for ListController.
+     *
+     * @param listInteractor the interactor for list-related use cases
+     * @param removeInteractor the interactor for remove-related use cases
+     */
+    public ListController(ListInputBoundary listInteractor, RemoveInputBoundary removeInteractor) {
```

Comment:
Like Simon mentioned, this can probably be separated into its own `ListRemoveController` to be compliant to the Single Responsibility Principle. It shouldn't be too bad to refactor this out either, you'd just need to create an `addMediaRemoveUseCase` or something like that in the AppBuilder (which he also mentioned above, I think).